warehouse/templates/packaging/detail.html

```diff
@@ -246,6 +246,11 @@ <h2 class="page-title">Download files</h2>
                     <a href="{{ request.route_url('packaging.file', path=file.path) }}">
                       {{ file.filename }}
                     </a>
+                    {% if file.has_signature %}
+                    <a href="{{ request.route_url('packaging.file', path=file.pgp_path) }}">
+                      <i class="fa fa-key" aria-hidden="true"></i>
```

Comment:
Might want to put `File Signature` to describe what this is. Just an icon is likely not sufficient.